x-pack/legacy/plugins/siem/cypress/integration/signal_detection_rules_custom.spec.ts

```diff
@@ -192,3 +196,67 @@ describe('Signal detection rules, custom', () => {
       .should('eql', '1m');
   });
 });
+
+describe('Deletes custom rules', () => {
+  beforeEach(() => {
+    esArchiverLoad('custom_rules');
+    loginAndWaitForPageWithoutDateRange(DETECTIONS);
+    waitForSignalsPanelToBeLoaded();
+    waitForSignalsIndexToBeCreated();
+    goToManageSignalDetectionRules();
+  });
+
+  after(() => {
+    esArchiverUnload('custom_rules');
+  });
+
+  it('Deletes one rule', () => {
+    cy.get(RULES_TABLE)
+      .find(RULES_ROW)
+      .then(rules => {
+        const initialNumberOfRules = rules.length;
+        const expectedNumberOfRulesAfterDeletion = initialNumberOfRules - 1;
+
+        cy.get(SHOWING_RULES_TEXT)
+          .invoke('text')
+          .should('eql', `Showing ${initialNumberOfRules} rules`);
+
+        deleteFirstRule();
```

Comment:
What do you think about adding assertions for the toaster messages that follow these actions? I think it would be nice to test that user feedback at this level.

Reply:
Makes sense! Thx for the suggestion 😊 

I'll merge this PR and I'll add to my automation list the toaster checks.